server/discord.py

```diff
@@ -11,6 +12,20 @@
 	titleDatabase = t[0]
 	titlesToUID = t[1]
 
+class NetworkIDsToName(Enum):
```

Comment:
Can we define this enum in a separate file and import it so that it doesn’t need to be redefined between files?

Reply:
Moved it to a new file called networks.py